build.gradle

```diff
@@ -158,16 +158,15 @@ task sourcesJar(type: Jar, dependsOn: generateSources) {
     from sourceSets.main.resources
 }
 
-artifacts {
-   archives shadowJar
-   archives libJar
-   archives sourcesJar
-}
-
 publishing {
     publications {
         mavenJava(MavenPublication) {
-            artifact jar
+            from components.java
+            suppressPomMetadataWarningsFor "runtimeElements" // Suppress errors about versions with '+' wildcard
```

Comment:
Id move away from using + versions tbh.

Reply:
Yep, the `+` for tiny remapper dep in yarn buildscript caused some troubles with 1.15.2 when tiny remapper updated asm to 8.

Reply:
Okay, I removed them from all `implementation` deps (and removed some duplicated deps as well).